lib/modules/manager/composer/artifacts.spec.ts

```diff
@@ -92,14 +92,17 @@ describe('modules/manager/composer/artifacts', () => {
     expect(
       await composer.updateArtifacts({
         packageFileName: 'composer.json',
-        updatedDeps: [{ depName: 'foo' }, { depName: 'bar' }],
+        updatedDeps: [
+          { depName: 'foo', newVersion: '1.0.0' },
```

Comment:
do we still have a test, where `newVersion` isn't set? so it can test current behavior?

Reply:
Yes, the `'disable plugins when configured locally'` one